mlir/lib/Dialect/GPU/Pipelines/GPUToNVVMPipeline.cpp

```diff
@@ -73,7 +73,6 @@ void buildCommonPassPipeline(
 //===----------------------------------------------------------------------===//
 void buildGpuPassPipeline(OpPassManager &pm,
                           const mlir::gpu::GPUToNVVMPipelineOptions &options) {
-  pm.addNestedPass<gpu::GPUModuleOp>(createStripDebugInfoPass());
```

Comment:
Why was this added in the first place?

Reply:
Not sure. Looking at the git history (https://reviews.llvm.org/D155463), there was a test case that had it in the pass pipeline. See also https://reviews.llvm.org/D103187. Maybe @ThomasRaoux knows.

Reply:
I remember that passing ptx with debug info used to crash nvidia driver. I had not debugged it at the time though

Reply:
Good to know. Given that the `printf` integration test works on my machine, this seems to be resolved.